npm-intro/npm-intro-quiz-notes.md

```diff
@@ -0,0 +1,49 @@
+# npm-intro-quiz-notes
+
+## Quiz Questions
+
+After completing this exercise, you should be able to discuss or answer the following questions:
+
+- What is NPM?
+
+- What is a package?
+
+- What are some other popular package managers?
+
+- How can you create a `package.json` with `npm`?
+
+- What is a dependency and how do you add one to a package?
+
+- What happens when you add a dependency to a package with `npm`?
+
+- What is a devDependency and how do you add one to a package?
+
+- How do you define and run `npm` scripts? Why are these useful?
```

Comment:
Don't forget the quiz questions!

Reply:
Sorry about that, all good now!

![Screenshot 2023-06-21 at 10 38 42 AM](https://github.com/j-alejandro-araujo/c0523-code-solutions/assets/91649194/eb5050ae-dbe0-4cbb-81a9-4168c57d1a15)